content/en/storage-providers/operate/snap-deals/index.md

```diff
@@ -129,43 +130,53 @@ Now that you have created a basic CC sector, it's time to convert it to a snap-d
 
     This command does not output anything on success.
 
-1. By listing your deals again, you'll see that the FSM has marked the sector as being in a `SnapDealsWaitDeals` state:
+1. By listing your deals again, you'll see that the FSM has marked the sector as being in a `Available` state:
 
     ```shell
     lotus-miner sectors list
 
     > ID  State                 OnChain  Active  Expiration                   Deals  DealWeight  VerifiedPower
     > 0   Proving               YES      YES     1550097 (in 10 weeks 1 day)  CC
     > 1   Proving               YES      YES     1550097 (in 10 weeks 1 day)  1      2KiB        18KiB
-    > 2   SnapDealsWaitDeals    YES      YES     1549517 (in 10 weeks 1 day)  CC
+    > 2   Available    YES      YES     1549517 (in 10 weeks 1 day)  CC
     ```
 
     This means that this sector (`2`) is ready to wait for deals!
 
 While the sector is transitioning through the snap-deals states, this sector is still preserved, and nothing is changing internally. The FSM works to keep the sector safe so that WindowPosts and WinningPosts are totally undisrupted during the process.  
 
-### Add data to the snap-deal sector
+### Accept the new storage deal
 
-1. Create a `UUID` variable and set it to the result of `uuidgen`:
+After the CC sector has been converted to accept the snap-deal, a new storage deal is accepted the storage provider will be sealed into the converted sector. You can track the status by checking the sector logs using the below command.
```

Comment:
>converted to accept the snap-deal, a new storage deal is accepted the storage provider will be sealed into the converted sector.

Maybe change to:
`converted to accept the snap-deal and a new storage deal is accepted, the storage provider will seal it into the converted sector.`

Reply:
@rjan90 Can you check this again as I merged the suggestions from @TippyFlitsUK .  This is outdated now.

Reply:
LGTM. Resolving